__tests__/integration/create-nft.test.js

```diff
@@ -0,0 +1,183 @@
+import { AUTHORITY_VALUE, TestUtils, TOKEN_DATA } from './utils/test-utils-integration';
+import { WalletHelper } from './utils/wallet-helper';
+
+describe('create-nft routes', () => {
+  /** @type WalletHelper */
+  let wallet1;
+  let wallet2;
+
+  const nftData = {
+    name: 'Test NFT',
+    symbol: 'TNFT',
+    data: 'ipfs://bafybeigu4pynylpbces2eq6hw6kflpjuagqkbqpawa7hrdnm5stzpyuwwe/'
+  };
+
+  beforeAll(async () => {
+    wallet1 = new WalletHelper('create-nft-1');
+    wallet2 = new WalletHelper('create-nft-2');
+
+    await WalletHelper.startMultipleWalletsForTest([wallet1, wallet2]);
+
+    await wallet1.injectFunds(1010);
+  });
+
+  it('should reject without the mandatory parameters', async done => {
+    for (const field of ['name', 'symbol', 'amount', 'data']) {
+      const token = {
+        ...nftData,
+        amount: 1000,
+        dontLogErrors: true
+      };
+      delete token[field];
+      const nftErr = await wallet1.createNft(token)
+        .catch(err => err);
+
+      expect(nftErr).toBeInstanceOf(Error);
```

Comment:
`expect.rejects` works best when the only assertion to be made about the error is about its string `message`.

In our case, we needed to make several assertions about the error object, and for that we needed this `catch` returning an error, plus asserting that the object was indeed of the `Error` type.

But since we've refactored these tests to make the http request directly, this helper code was removed.